package.json

```diff
@@ -17,7 +17,7 @@
     "/oclif.manifest.json"
   ],
   "scripts": {
-    "build": "shx rm -rf dist && tsc -b && npm test",
+    "build": "shx rm -rf dist && tsc -b && shx cp -r src/resources dist/resources && npm test",
```

Comment:
I added this `cp` command to copy static template files (`Dockerfile` and `.gitignore`) to the `dist` directory. Not sure if it is the best way to handle this.

Reply:
I think we may need a separated script to copy resources. @Dv92 any hint on this?

Reply:
yes, I agree you can make an isolated command

```json
{
  "scripts": {
    "copy:resources": "shx cp -r src/resources dist/resources",
    "build": "shx rm -rf dist && tsc -b && npm run copy:resources && npm test"
  }
}
```

maybe it's better to move outside of `src` the resources dir if this dir contains only static files